### ISSUE TITLE
About VCARD

### ISSUE THREAD

**Issue by diegozl** [open-wa/wa-automate-nodejs#2836]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Current Behavior

Hi,

I was able to make the contact appear, as you guided. Thanks! My only question is: Is there a way to define the name on the sticker? Here for me, it is "contact".

Currently, my code is like this:
`await client.sendVCard(message.from, "BEGIN:VCARD\r\nVERSION: 3.0\r\nN:LastName;FirstName;;;\r\nFN:FistName LastName\r\nORG:Business;\r\nTEL;waid=551111111111:+55 11 1111-1111\r\nEND:VCARD", 'New Contact');`

Does anyone know how to proceed?

### Steps To Reproduce

Just run.

### DEBUG INFO

```markdown
WA_VERSION: WhatsApp Business
```

### Environment

```markdown
- OS: Windows_NT x64
- Node: 16.13.2
- npm:
```

**Comment by smashah** [comment]
Hi, your vcard is not valid. Something like this will work:

```javascript
await client.sendVCard(id, `BEGIN:VCARD
VERSION:3.0
N:Lastname;FirstName
FN:FirstName LastName
ORG:Business
URL:http://www.example.com/
EMAIL:info@example.com
END:VCARD`, 'New Contact', '551111111111')
```

**Comment by diegozl** [comment]
> Hi, your vcard is not valid. Something like this will work:
> 
> ```js
> await client.sendVCard(id, `BEGIN:VCARD
> VERSION:3.0
> N:Lastname;FirstName
> FN:FirstName LastName
> ORG:Business
> URL:http://www.example.com/
> EMAIL:info@example.com
> END:VCARD`, 'New Contact', '551111111111')
> ```

Hi, thank you for faster response. 

First, congratulations on the project. It's amazing!

But the method you sent, oddly enough, gives an error, here, it doesn't pull the image (since it doesn't have the waid) and it doesn't pull the data.

I did as follows:
1 - I configured it so, whenever the bot receives a message and doesn't know how to respond, it sends it to me:
`await client.sendText('mynumber', message.body);`
2 - I uploaded the VCARD the way I wanted and copied the code :)
3 - Unfortunately, the only problem is that it doesn't pull the contact, but pulls everything else, such as logo, company map, opening hours, etc., and generates the "Chat with" and "View company" buttons :D
